### PR TITLE
Group Role Binding Support. This closes #43

### LINKED ISSUE #43: Role_binding support for add-role-to-group
the role_binding in `ansible-stacks` right now only supports `add-role-to-user` and not `add-role-to-group` 

See corresponding issue: https://github.com/rht-labs/api-design/issues/31

### PR DESCRIPTION
**Pull request by mcanoy**
#### What does this PR do?

closes #43 . Adds the ability to assign roles by group.
#### Which tests illustrate how this code works?

cdk_create_labs_automation_api.yml
#### Is there a relevant Issue open for this?
#43
#### Are there any other relevant resources that should be reviewed as well?

n/a
#### Who would you like to review this?

/cc @oybed @sherl0cks

### CODE REVIEW

File: roles/create-openshift-resources/tasks/create_project.yml [hunk 1]
```diff
@@ -58,6 +64,16 @@
   when: role_bindings_present == true
   static: no
 
+
+- debug: msg="group role binding pres {{ group_role_bindings_present }} {{ project}}"
+
```
Comment (sherl0cks) [inline roles/create-openshift-resources/tasks/create_project.yml]: Please remove this

**Comment by sherl0cks** [comment]
@mcanoy looks good but I need to see that a test has been updated to use this syntax. can you update the `cdk_create_labs_automation_api.yml` test to use this role please and verify it works on the CDK?

**Comment by mcanoy** [comment]
@sherl0cks You are correct. I omitted the change I made to test/vars/automation_api.json. The added group_to_role element will cause the `cdk_create_labs_automation_api.yml` test invoke this task for labs-api-dev and it will be skipped in the labs-api-delivery. I've added the commit to show this.

**Comment by oybed** [comment]
LGTM - thanks for making this change.

**Comment by sherl0cks** [comment]
@mcanoy thanks for updates. lgtm

**Pull request merged**